components/developer/valgrind/valgrind.p5m

```diff
@@ -48,18 +48,125 @@ file path=usr/bin/$(MACH64)/valgrind
 file path=usr/bin/$(MACH64)/valgrind-di-server
 file path=usr/bin/$(MACH64)/valgrind-listener
 file path=usr/bin/$(MACH64)/vgdb
-
 file path=usr/include/valgrind/callgrind.h
+file path=usr/include/valgrind/config.h
```

Comment:
Comment on upper part of valgrind.p5m : what is the point of having isaexec symlinks an delivering only 64-bit version? I'd dropped them and delivered just 64-bit binaries in /usr/bin . 'CONFIGURE_BINDIR.64 = $(CONFIGURE_PREFIX)/bin' can help.